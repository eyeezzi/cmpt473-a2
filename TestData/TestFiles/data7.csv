firstname, lastname
"Don,ald", "Knuth"
"John", "Doe"