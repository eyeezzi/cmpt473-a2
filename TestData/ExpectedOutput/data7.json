[
	{
		"firstname": "Don,ald",
		" lastname": " \"Knuth\""
	},
	{
		"firstname": "John",
		" lastname": " \"Doe\""
	}
]